### PR TITLE
(AzureCXP) Added more detail

### PR DESCRIPTION
**Pull request by BhargaviAnnadevara-MSFT**
Added more detail to help find the setting.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@GitHubber17 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/library/service-endpoints.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ use one of the following techniques:
 
 - If you want to authorize any pipeline to use the service connection,
   go to Azure Pipelines, open the Settings page, select Service connections,
-  and enable the setting **Allow all pipelines to use this connection** option for the connection.
+  and enable the setting **Allow all pipelines to use this connection** option for the connection, found under the **Policies** tab.
```
Comment (RoopeshNair) [inline docs/pipelines/library/service-endpoints.md]: Thank you for your contribution, the service connections UI has been overhauled, you can enable the new UX from preview features. New experience has this setting as part of security config. We have to re-do this.
image

**Comment by Jak-MS** [comment]
@RoopeshNair - can you clarify what you'd like to do with this PR? Are you still waiting for @BhargaviAnnadevara-MSFT  to make changes? Or should this be closed? Thanks.

**Comment by RoopeshNair** [comment]
@Jak-MS  - I believe we can close this PR. @ashokirla / @shashban  for updates to authorization experience.

File: docs/pipelines/library/service-endpoints.md (hunk 1 above)
Comment (BhargaviAnnadevara-MSFT) [reply docs/pipelines/library/service-endpoints.md]: @RoopeshNair I see. Thank you for that tip!

**Pull request closed without merging**